modules/fileout_netcdf/FONcArray.cc

```diff
@@ -61,6 +61,12 @@ vector<FONcDim *> FONcArray::Dimensions;
 
 const int MAX_CHUNK_SIZE = 1024;
 
+// Set general total maximum chunk sizes to 1M(1024x1024)
+const int GENERAL_MAX_CHUNK_SIZES = 1048576; 
```

Comment:
I was wondering, do we ever want to make chunks as small as 1K elements?

Reply:
We've done this all the time for 1-D array, even smaller than 1K if the array size is small. It doesn't matter too much for me since it doesn't hurt performance too much. The real issue we solved in this pull request is when the 1-D array is very big , we need to increase the chunk size.